faucet/port.py

```diff
@@ -107,6 +107,8 @@ class Port(Conf):
         # Min time since last LACP response. Used to control rate of response for LACP
         'lacp_port_priority': 255,
         # Sets port priority value sent out in lacp packet
+        'lacp_port_id': 0,
```

Comment:
I think 0 could be a valid lacp_port_id value. Unsure if setting this to None will work with it being an `int` type in our config parser. Maybe -1 should be the default value that indicates faucet should autogen the value?

Reply:
Reset default value to -1.